jaxlib/_jax/__init__.pyi

```diff
@@ -928,6 +928,14 @@ class Client:
       host_callbacks: Sequence[typing_extensions.CapsuleType] = ...,
   ) -> LoadedExecutable: ...
   @overload
+  def deserialize_executable(
+      self,
+      serialized: bytes,
+      executable_devices: DeviceList,
+      compile_options: CompileOptions | None = None,
+      host_callbacks: Sequence[Callable[..., Any]] = [],
```

Comment:
One last thing -- could you please set the default here to `...` instead of `[]`?

Reply:
Done.